Listener/CommandListener.php

```diff
@@ -47,15 +47,9 @@ public static function getSubscribedEvents()
     {
         $events = array(
             ConsoleEvents::COMMAND => array('onConsoleCommand', 0),
+            ConsoleEvents::ERROR => array('onConsoleError', 0),
         );
 
-        if (class_exists('Symfony\Component\Console\Event\ConsoleErrorEvent')) {
-            $events[ConsoleEvents::ERROR] = array('onConsoleError', 0);
-        } else {
-            // backward compatibility with Symfony < 3.3
-            $events[ConsoleEvents::EXCEPTION] = array('onConsoleException', 0);
```

Comment:
are we dropping support of symfony < 4? If yes you should update composer.json

Reply:
We are dropping support for Symfony <3.4. I've updated composer.json in this PR: https://github.com/ekino/EkinoNewRelicBundle/pull/138/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R16